holoviews/plotting/util.py

```diff
@@ -130,23 +130,26 @@ def compute_overlayable_zorders(obj, path=[]):
     # Process the inputs of the DynamicMap callback
     dmap_inputs = obj.callback.inputs if obj.callback.link_inputs else []
     for z, inp in enumerate(dmap_inputs):
+        no_increment = False
```

Comment:
Should this be called ``no_z_increment``?

Reply:
Or ``zorder_unchanged``?

Reply:
Yes, sure.